luigi/contrib/kubernetes.py

```diff
@@ -42,74 +44,46 @@
 logger = logging.getLogger('luigi-interface')
 
 try:
-    from pykube.config import KubeConfig
-    from pykube.http import HTTPClient
-    from pykube.objects import Job, Pod
-except ImportError:
-    logger.warning('pykube is not installed. KubernetesJobTask requires pykube.')
+    import kubernetes as kubernetes_api
+except ImportError as i:
+    logger.warning("WARNING: kubernetes is not installed. KubernetesJobTask requires kubernetes")
+    logger.warning("  Please run 'pip install kubernetes' and try again")
 
 
 class kubernetes(luigi.Config):
-    auth_method = luigi.Parameter(
-        default="kubeconfig",
-        description="Authorization method to access the cluster")
-    kubeconfig_path = luigi.Parameter(
-        default="~/.kube/config",
-        description="Path to kubeconfig file for cluster authentication")
-    max_retrials = luigi.IntParameter(
-        default=0,
-        description="Max retrials in event of job failure")
```

Comment:
A note for reviewers.  These things are safe to remove, and no longer need to live in code, as the upstream Kubernetes library supports handling them automatically via Environment Variables.  The max_retrials also (unless I misunderstand) is completely unnecessary as it duplicates functionality with `backoff_limit`.  Arguably, one config option here I could keep is auth_method so an end-user can "force" which auth method to use.  But, the Kubernetes library should be able to automatically detect and cascade into the "mode" it needs to function.